scripts/monitoring/cron-send-docker-metrics.py

```diff
@@ -46,8 +50,54 @@
         }
 
     zs.add_zabbix_keys(keys)
-
     print "Sending these metrics:"
     print json.dumps(keys, indent=4)
     zs.send_metrics()
     print "\nDone.\n"
+    #start auto-heal
+
+    if int(du_dds.data_space_percent_available) < 90:
+        print 'Docker has less than 90% storage avaiable. Attempting to clean up space.'
+        print '***********************************'
+        #clean the exited containers
+        for container in exited_containers:
+            print '*******start cleaning**************'
+            print 'the container id is :', container['Id']
+            #get the info
+            exited_container = cli.containers(all=True, filters={'id':container['Id']})
+            #print Deadcontainer
+            if len(exited_container) == 1:
+                print 'the status of the container is :', exited_container[0]['Status']
+                if exited_container[0]['Status'].find('Exited') != -1:
+                    #do the remove step
+                    print 'status confirmed :', exited_container[0]['Status']
+                    try:
+                        cli.remove_container(container=container['Id'])
+                        print 'done this container'
+                    except Exception, e:
+                        print 'something wrong during the remote of the container'
+            else:
+                print 'container not exist'
+        #clean the dead containers
+        for container in dead_containers:
+            print '*******start cleaning**************'
+            print 'the container id is :', container['Id']
+            #get the info
+            dead_container = cli.containers(all=True, filters={'id':container['Id']})
+            #print Deadcontainer
+            if len(dead_container) == 1:
+                print 'the status of the container is :', dead_container[0]['Status']
+                if dead_container[0]['Status'].find('Dead') != -1:
+                    #do the remove step
+                    print 'status confirmed :', dead_container[0]['Status']
+                    try:
+                        cli.remove_container(container=container['Id'])
+                        print 'done this container'
+                    except Exception, e:
+                        print 'something wrong during the remote of the container'
+
+            else:
+                print 'container not exist'
+    else:
+        print 'Docker storage has more than 90% available. Skipping autoheal to clean up space', int(du_dds.data_space_percent_available)
```

Comment:
----
      print` 'Docker storage has more than 90% available. Skipping autoheal to clean up space', 
           int(du_dds.data_space_percent_available)
----